packages/web-components/test/agoric-wallet-connection.test.js

```diff
@@ -76,8 +76,8 @@ describe('AgoricWalletConnection', () => {
   });
 
   it(`has a default state of 'idle' and has a walletConnection`, async () => {
-    const onState = ev => {
-      expect(ev.detail.state).to.equal('idle');
+    const onState = async ev => {
+      await expect(ev.detail.state).to.equal('idle');
```

Comment:
more concise to return the promise:
```suggestion
    const onState = ev => expect(ev.detail.state).to.equal('idle');
```

I'd opt for the more verbose syntax if the types were't apparent, but we do have the linter to inform about unhandled promises

Reply:
Done